models/business-unit/src/associate/generator.ts

```diff
@@ -0,0 +1,15 @@
+import { Reference } from '@commercetools-test-data/commons';
+import { fake, Generator } from '@commercetools-test-data/core';
+import * as AssociateRoleAssignment from '../associate-role-assignment';
+import { TAssociateDefault } from './types';
+
+// https://docs.commercetools.com/api/projects/business-units#associate
+
+const generator = Generator<TAssociateDefault>({
+  fields: {
+    associateRoleAssignments: [fake(() => AssociateRoleAssignment.random())],
```

Comment:
```suggestion
    associateRoleAssignments: fake(() => [AssociateRoleAssignment.random()]),
```

Reply:
fixed in [0c72cd8](https://github.com/commercetools/test-data/pull/481/commits/0c72cd8436aafea8d4e2897fc78208e5bf5ccf05)